packages/browser-repl/src/components/shell-input.tsx

```diff
@@ -71,9 +73,9 @@ export class ShellInput extends Component<ShellInputProps, ShellInputState> {
       return;
     }
 
-    this.setState({
-      currentValue: value,
-      didLoadHistoryItem: true
+    this.setState({ currentValue: value }, () => {
+      // eslint-disable-next-line chai-friendly/no-unused-expressions
+      this.editor?.navigateFileEnd();
```

Comment:
That's the only "functional" change in this PR: it always bothered me how we move cursor to the end of the line using state and props, technically this is just a side-effect of the state update, so makes sense to me to handle it like one